examples/imported_function.py

```diff
@@ -0,0 +1,20 @@
+from wasmer import Instance
+import os
+
+__dir__ = os.path.dirname(os.path.realpath(__file__))
+
+wasm_bytes = open(__dir__ + '/imported_function.wasm', 'rb').read()
+
+def sum(x: 'i32', y: 'i32') -> 'i32':
```

Comment:
I don't think the typings should be mandatory here. And I also rather have native python types rather than strings, since it will play much nicer with tools like `mypy`.

```
def sum(x: int, y: int) -> int:
```

However can already know the types given the imported function signature, so perhaps is better to let python types to python itself. and get the function signature from the module and cast it to what we need.

Reply:
Thinking more, the strategy currently used will also not work with imports with dynamic arguments:

```python
def sum (*args):
  sum(args)
```

Reply:
Of course variadic functions aren't supported as host functions. It doesn't even not work in our runtime :-p.

Reply:
Thinking of it, it might be actually possible in Python to support variadic host functions.

Reply:
It requires however to compile the module before instantiating it. Not a big deal though.

Reply:
Done.